SLM/events/Silvermont_core.json

```diff
@@ -905,7 +905,7 @@
       "UMask": "0x10",
       "EventName": "BACLEARS.COND",
       "BriefDescription": "Counts the number of JCC baclears",
-      "PublicDescription": "The BACLEARS event counts the number of times the front end is resteered, mainly when the Branch Prediction Unit cannot provide a correct prediction and this is corrected by the Branch Address Calculator at the front end.  The BACLEARS.COND event counts the number of JCC (Jump on Condtional Code) baclears.",
+      "PublicDescription": "The BACLEARS event counts the number of times the front end is resteered, mainly when the Branch Prediction Unit cannot provide a correct prediction and this is corrected by the Branch Address Calculator at the front end.  The BACLEARS.COND event counts the number of JCC (Jump on Conditional Code) baclears.",
```

Comment:
Looks good to me. Please increase the version in the header at the top of this file.